### PR TITLE
Bernardo's Pancake Recipe

### PR DESCRIPTION
**Pull request by jennank**
Adding second pancake recipe. Other random fixes also included for formatting.

### CODE REVIEW

**Comment by jennank** [comment]
Need to fix the servings for pancakes.

File: source/breakfast/index.rst [hunk 1]
```diff
@@ -4,4 +4,4 @@ Breakfast
 .. toctree::
    :maxdepth: 1
 
-   pancakes
+   jennaPancakes
```
Comment (bernardopacini) [inline source/breakfast/index.rst:8]: This is missing `bernardoPancakes` I think?

**Review by bernardopacini — CHANGES_REQUESTED** [review]

File: source/breakfast/index.rst (hunk 1 above)
Comment (jennank) [reply source/breakfast/index.rst:8]: Fixed.

**Review by bernardopacini — APPROVED** [review]
This looks good to me. Is this ready?

**Pull request merged**